src/oidentd.js

```diff
@@ -1,68 +1,42 @@
 "use strict";
 
-var fs = require("fs");
-var Helper = require("./helper");
+const fs = require("fs");
+const Helper = require("./helper");
 
 function OidentdFile(file) {
 	this.file = Helper.expandHome(file);
 	this.connectionId = 0;
-	this.connections = {};
+	this.connections = new Map();
 
 	this.refresh();
 }
 
 OidentdFile.prototype = {
-	hookSocket: function(socket, user) {
-		var that = this;
-		var id = null;
+	addSocket: function OidentdFileAddSocket(socket, user) {
```

Comment:
We definitely should start using classes, yeah. But for this PR I'll just remove the name additions.